spk/syncthing/Makefile

```diff
@@ -24,7 +24,6 @@ SERVICE_SETUP = src/service-setup.sh
 SERVICE_PORT = 8384
 SERVICE_PORT_TITLE = $(DISPLAY_NAME)
 SERVICE_CERT = syncthing_webui
-SERVICE_CERT_RELOAD = tools/ca_reloader.sh
```

Comment:
I think that variable is used under DSM 6 for setting up the certificate management.  I see why you removed it in the lines below, but for the other part we still need it.  Look in `spksrc.service.mk`, it's documented there.

Reply:
done in #5118